integration/docker-compose.pxe.yml

```diff
@@ -34,8 +34,9 @@ services:
   sztp:
     build:
       context: pxe
+      dockerfile: pxe/Dockerfile.sztp
     network_mode: service:pxe
-    command: ['sztpd', '--help']
+    command: ['sztpd', 'sqlite:///:memory:']
```

Comment:
Ah, the infamous `memory` backend. :)

Reply:
will change this once this starts working
```
  :memory:    - an in-memory database (only useful for testing)
  <filepath>  - an OS-specific filepath to a persisted database file

  Examples:

    $ sztpd sqlite:///:memory:                      (memory)
    $ sztpd sqlite:///relative/path/to/sztpd.db     (unix)
    $ sztpd sqlite:////absolute/path/to/sztpd.db    (unix)
    $ sztpd sqlite:///C:\path\to\sztpd.db           (windows)
```